packages/api/src/controllers/access-control.ts

```diff
@@ -31,18 +31,35 @@ accessControl.post(
       (await db.stream.getByPlaybackId(playbackId)) ||
       (await db.asset.getByPlaybackId(playbackId));
 
+    let contentLog;
+    let userLog;
+    let signingKeyLog;
+
     if (!content || content.deleted) {
+      try {
+        contentLog = JSON.stringify(content);
+      } catch (e) {
+        contentLog = content;
+      }
+
       console.log(`
-        access-control: gate: content not found for playbackId ${playbackId}, disallowing playback
+        access-control: gate: content not found for playbackId ${playbackId}, disallowing playback, content=${contentLog}
       `);
       throw new NotFoundError("Content not found");
     }
 
     const user = await db.user.get(content.userId);
 
     if (user.suspended || ("suspended" in content && content.suspended)) {
+      try {
+        contentLog = JSON.stringify(content);
+        userLog = JSON.stringify(user);
+      } catch (e) {
+        contentLog = content;
+      }
+
       console.log(`
-        access-control: disallowing access for contentId=${content.id}, user ${user.id} is suspended
+        access-control: gate: disallowing access for contentId=${content.id}, user ${user.id} is suspended, content=${contentLog}, user=${userLog}
```

Comment:
What about the playback ID in all logs?

Reply:
haven't added it because I was logging the full content, but yeah, better add it anyway so if the content is undefined for some reason we catch it